src/interfaces/special-resolution-interfaces/create-resolution-interface.ts

```diff
@@ -0,0 +1,9 @@
+import { SigningPersonIF } from '@/interfaces'
+
+export interface CreateResolutionIF {
```

Comment:
Since you are reusing this IF (hence its name from Create UI), please add a comment that this IF is also using for Special Resolution filings.